### ISSUE TITLE
Add JS on research gallery to calculate progress bar %

### ISSUE THREAD

**Issue by kbdonnally** [scholarslab/scholarslab.org#201]
### Context
1. What should the feature do?
- Right now the width of "you've seen this" and "you haven't seen this" are statically calculated, but want it to be calculated by how many items in the research gallery you've clicked through as a percent of total
- Items already have a `data-item-number` associated with them so it's just a matter of putting in an event listener on the arrow nav buttons that triggers a recalculation of widths every time you click
2. When and where should the feature appear?
- In `main.js` under research section, by demo day

### Associated Product Ticket
#130, #132

**Comment by amandavisconti** [comment]
This is cool, but I'm also interested in more of a grid layout like the photos on /people. Might make sense to wait on this issue's work until the demo?

**Comment by kbdonnally** [comment]
@amandavisconti Noted, and sounds good. I originally had it as a grid layout and changed it to a gallery after suggestions from some other team members (I forget who now), so probably best to make a version of both post-demo and decide which to go with then. Changing to Public Release milestone!

**Comment by amandavisconti** [comment]
Closing in favor of grid layout for now, but could return to horizontal scroll view (or use elsewhere, e.g. on touchscreen monitor in hallway, homepage, other page) post-public launch (new ticket #357 swaps back to grid for now)
